java-checks/src/test/files/checks/StringLiteralDuplicatedCheck.java

```diff
@@ -23,18 +23,22 @@ default void foo() {
     }
   }
 }
-
+class AllConstants {
+  private static final String Constant1 = "allConstant";
+  private static final String Constant2 = "allConstant";
+}
 class ConstantAlreadyDefined {
 
   private static final String A = "constant";
-  private static final String B = "constant"; // Noncompliant [[secondary=31,36]] {{Use already-defined constant 'A' instead of duplicating its value here.}}
+  private static final String B = "constant";
   private static final String C = "constant";
 
   private static final String REPORT_WITHOUT_THRESHOLD = "blabla";
 
   void test() {
+    System.out.println("constant"); // Noncompliant [[secondary=40]] {{Use already-defined constant 'A' instead of duplicating its value here.}}
     System.out.println("constant");
-    System.out.println("blabla"); // Noncompliant {{Use already-defined constant 'REPORT_WITHOUT_THRESHOLD' instead of duplicating its value here.}}
+    System.out.println("blabla");
```

Comment:
Why is this not reported anymore?  Condition on line 68 should be changed for `> 0`